File: src/OrtographicCamera.java
import javax.vecmath.*;

public class OrtographicCamera extends Camera {
	private double size = 0d;
	
	public OrtographicCamera(double size) {
		this.size = size;
	}
	
	public Ray generateRay(Point2d point) {
		// ToDo: Generate the ray by
		// providing a point and a vector
		Ray ray = new Ray(new Point3d(), new Vector3d(0d, 0d, 1d));
		return ray;
	}
}

File: src/OrtographicCamera.scala
import javax.vecmath._;

case class OrtographicCamera ( size:Double ) 
{
  def generateRay ( point:Point2d ):Ray = 
    {    
     
      var ray = new Ray ( new Point3d(point.getX(),point.getY(),0), new Vector3d(0d,0d,1d));
      return ray;
    }
  
}
